lists/ir.csv

```diff
@@ -1,9 +1,6 @@
 url,category_code,category_description,date_added,source,notes
-http://213.4.130.210,NEWS,News Media,2014-04-15,citizenlab,

```

Comment:
This site is safe to delete as it doesn't appear to be blocked based on OONI data.